profile_page/index.php

```diff
@@ -0,0 +1,86 @@
+<?php
+// Validate is user logged in
+require_once(__DIR__ . '/../validate_user.php');
```

Comment:
This also validates if a user is admin. I would decouple the two and only validate if the user is logged in